README.md

```diff
@@ -214,3 +215,5 @@ Want to add yours? Awesome! Send a PR with your link added to the bottom of the
 [fsu-beamer-theme-pdf]: https://github.com/llamabr/fsu-beamer-theme/blob/master/fsu-beamer-theme.pdf
 [fit-beamer-theme]: https://github.com/llamabr/fit-beamer-theme
 [fit-beamer-theme-pdf]: https://github.com/llamabr/fit-beamer-theme/blob/master/fit-beamer-theme.pdf
+[UT-LaTeX-Beamer-Theme]: https://www.utwente.nl/en/eemcs/sor/staff/goseling/utbeamer/
+[ut-latex-beamer-theme]: https://www.utwente.nl/en/eemcs/sor/staff/goseling/utbeamer/res/manual.pdf
```

Comment:
If you check out the preview of this file in GitHub, this reference is resolving to L218. This could possibly be due to the markdown parser interpreting references as case insenstivie. Can you try adding the `-pdf` postfix to this ref and seeing if that works? Cheers!